{"name": "test_getFundAccount_HK", "status": "broken", "statusDetails": {"message": "NameError: name 're' is not defined", "trace": "self = <TestCase.SimulatedStock.test_getFundAccount.TestgetFundAccount object at 0x0000024FC5E62B50>\n\n    @allure.story('查询资金账户接口_1-港股')\n    def test_getFundAccount_HK(self):\n        q = OperationSql(\"192.168.1.237\", \"root\", \"123456\", \"user_account\")\n        userId = q.show_sql(\"select id from t_user_account where `zr_no`= '68904140';\")\n        # print(userId)\n        response = zhuorui('模拟炒股', '查询资金账户接口_1-港股')\n        assert response.status_code == 200\n        assert_data(response, '000000', 'ok')\n        # print(response.json())\n        if \"data\" in response.json():\n            if len(response.json().get(\"data\")) != 0:\n>               assert response.json().get(\"data\").get(\"userId\") == re.search(\"(\"\",)\", userId)\nE               NameError: name 're' is not defined\n\ntest_getFundAccount.py:35: NameError"}, "start": 1597225752306, "stop": 1597225752375, "uuid": "52b3c117-70f3-4235-a1be-b4f31a065f29", "historyId": "4d609e8d0492c262fd610331faa2db43", "testCaseId": "c859460baf8af3f84425cdb2909f672b", "fullName": "TestCase.SimulatedStock.test_getFundAccount.TestgetFundAccount#test_getFundAccount_HK", "labels": [{"name": "story", "value": "查询资金账户接口_1-港股"}, {"name": "parentSuite", "value": "TestCase.SimulatedStock"}, {"name": "suite", "value": "test_getFundAccount"}, {"name": "subSuite", "value": "TestgetFundAccount"}, {"name": "host", "value": "DESKTOP-KBU1G8M"}, {"name": "thread", "value": "86244-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCase.SimulatedStock.test_getFundAccount"}]}